resources/digital_systems.md

```diff
@@ -0,0 +1,226 @@
+
+```
+Codes = Group of Symbols
+```
+
+Binary Codes can described as being part of one or more of the following
+families:
+
+| Weighted Codes                                                                                    | Non-Weighted Codes                                                                                   | Reflective Codes                                                                                           | Sequential Codes                                                                                                            | Alphanumeric Codes                                                                                                  | Error Detecting & Corrector Codes                                                                                         |
+|:--------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| There is a weight associated with each digit, dependant on the number's position in the sequence. | Digit Value is independent of a number's position in the sequence. Gray Code is one example of this. | _k-th_ smallest number is the complement of the _K-th_ largest number. Look up the 5211 code or 2421 code. | For every subsequent number, the binary representation only differs by one digit. Excess-3 and 8421 are codes of this type. | The family of binary codes used to represent alphanumeric data. The primordial example would be the ASCII encoding. | The family of binary encoding is meant to detect and or correct failed data transmission. Parity Checking is one example. |
+
+---
+
+## Multiplexer
+
+A multiplexer is a combinational circuit that selects one of many input lines
+and directs it to a single output line.
+The selection of a particular input line is controlled by a set of selection
+lines. Normally, there are $2^𝑛$ input
+lines and 𝑛 selection lines whose bit combinations determine which input is
+selected.
+
+An example of a 4 to 1 multiplexer is provided below.
+
+![4 to 1 Multiplexer.png](/resources/pictures/4%20to%201%20Multiplexer.png)
+
+---
+
+## Demuxes
+
+> Basic Multiplexer System. Permission allowed since attribution is fulfilled.
+> ![Telephony_multiplexer_system.gif](/resources/pictures/Telephony_multiplexer_system.gif)
+> Tony R. Kuphaldt, CC BY 1.0 <https://creativecommons.org/licenses/by/1.0>, via
+> Wikimedia Commons
```

Comment:
You don't need to write out the full attribution. See previous review, I was suggesting we use the direct website link to the image. So in this case  it would be something like:

```markdown
![Telephone multiplexer gif](https://upload.wikimedia.org/wikipedia/commons/e/e0/Telephony_multiplexer_system.gif?20060313201122)
```

This way we maintain record to the direct image source and simplify files.

Reply:
The full attribution is necessary as part of the CC1.0 license. Even If I link to where I found the image, that would still be incorrect. The  place where I found it provedes attribution, but if I was to use it, I'd need to privide attribution of my own

Reply:
I could remove the block: "permission allowed since attribution is fulfilled." This would make the overall markdown cleaner.